performance/performance_tuning.rst

```diff
@@ -31,57 +31,59 @@ To begin the tuning process create a Citus cluster and load data in it. From the
 
 Here is an example of explaining the plan for a particular example query.
 
+.. code-block:: postgresql
+
+  EXPLAIN
+   SELECT date_trunc('minute', created_at) AS minute,
+          sum((payload->>'distinct_size')::int) AS num_commits
+     FROM github_events
+    WHERE event_type = 'PushEvent'
+    GROUP BY minute
+    ORDER BY minute;
+
 ::
 
-  explain SELECT comment FROM wikipedia_changes c, wikipedia_editors e
-          WHERE c.editor = e.editor AND e.bot IS true LIMIT 10;
-
-  Distributed Query into pg_merge_job_0005
-    Executor: Real-Time
-    Task Count: 16
-    Tasks Shown: One of 16
-    ->  Task
-      Node: host=localhost port=9701 dbname=postgres
-      ->  Limit  (cost=0.15..6.87 rows=10 width=32)
-        ->  Nested Loop  (cost=0.15..131.12 rows=195 width=32)
-          ->  Seq Scan on wikipedia_changes_102024 c  (cost=0.00..13.90 rows=390 width=64)
-          ->  Index Scan using wikipedia_editors_editor_key_102008 on wikipedia_editors_102008 e  (cost=0.15..0.29 rows=1 width=32)
-            Index Cond: (editor = c.editor)
-            Filter: (bot IS TRUE)
-  Master Query
-    ->  Limit  (cost=0.00..0.00 rows=0 width=0)
-      ->  Seq Scan on pg_merge_job_0005  (cost=0.00..0.00 rows=0 width=0)
-  (15 rows)
-
-This tells you several things. To begin with there are sixteen shards, and we're using the real-time Citus executor setting:
+  Sort  (cost=0.00..0.00 rows=0 width=0)
+    Sort Key: minute
+    ->  HashAggregate  (cost=0.00..0.00 rows=0 width=0)
+      Group Key: minute
+      ->  Custom Scan (Citus Real-Time)  (cost=0.00..0.00 rows=0 width=0)
+        Task Count: 32
+        Tasks Shown: One of 32
+        ->  Task
+          Node: host=localhost port=5433 dbname=postgres
+          ->  HashAggregate  (cost=93.42..98.36 rows=395 width=16)
+            Group Key: date_trunc('minute'::text, created_at)
+            ->  Seq Scan on github_events_102042 github_events  (cost=0.00..88.20 rows=418 width=503)
+              Filter: (event_type = 'PushEvent'::text)
+  (13 rows)
+
+This tells you several things. To begin with there are thirty-two shards, and we're using the real-time Citus executor setting:
```

Comment:
Instead of "we're using the real-time Citus executor setting", to maybe "the planner chose the Citus real-time executor to execute this query".